packages/react/src/infrastructure/solana-utils/SignTransaction.ts

```diff
@@ -0,0 +1,64 @@
+import { AnchorWallet } from '@solana/wallet-adapter-react';
+import { Keypair, Transaction } from '@solana/web3.js';
+import nacl from 'tweetnacl';
+
+export async function signTransaction(
+  pTransaction: Transaction,
+  wallet: AnchorWallet,
+  otherSigner?: Keypair
+): Promise<string> {
+  try {
+    const transaction: Transaction = pTransaction;
+    const signedTransaction = await wallet.signTransaction(transaction);
+    if (otherSigner) {
+      const transactionDataToSign = signedTransaction.serializeMessage();
+
+      const otherSignature = nacl.sign.detached(
+        transactionDataToSign,
+        otherSigner.secretKey
+      );
+      signedTransaction.addSignature(
+        otherSigner.publicKey,
+        Buffer.from(otherSignature)
+      );
+    }
+    return JSON.stringify(signedTransaction.serialize());
+  } catch (error) {
+    return '';
+  }
+}
+
+export async function signSwapTransactions(
+  swappedTransaction: Transaction,
+  standardTransaction: Transaction,
+  wallet: AnchorWallet,
+  otherSigner?: Keypair
+): Promise<{
+  swapSignedTx: string;
+  standardSignedTx: string;
+}> {
+  try {
+    const signedTransactions = await wallet.signAllTransactions([
```

Comment:
can you please destructure the sited transactions it's cleaner that way, it's also good to do it here since it will go into SDK:
```TS
    const [signedSwapTx, signedHelioTx] = await wallet.signAllTransactions()
```